addons/hr_payroll/report/payslip_details_report.py

```diff
@@ -94,9 +87,17 @@ def get_lines_by_contribution_register(self, obj):
                 })
         return res
 
-
-class wrapped_report_payslipdetails(osv.AbstractModel):
-    _name = 'report.hr_payroll.report_payslipdetails'
-    _inherit = 'report.abstract_report'
-    _template = 'hr_payroll.report_payslipdetails'
-    _wrapped_report_class = payslip_details_report
+    @api.multi
+    def render_html(self, data=None):
+        Report = self.env['report']
+        report = Report._get_report_from_name('hr_payroll.report_payslipdetails')
+        payslip = self.env['hr.payslip'].browse(self.env.context.get('active_ids'))
+        docargs = {
+            'doc_ids': self.ids,
```

Comment:
here you have to pass context active_ids instead of self.ids in all report... current model ids. otherwise report will be wrong..